plugins/file-open-save-new/core/src/test/java/org/pentaho/di/plugins/fileopensave/providers/vfs/VFSFileProviderTest.java

```diff
@@ -24,96 +24,16 @@
 
 import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
 
 public class VFSFileProviderTest extends TestCase {
 
-  public void testIsConnectionRoot() {
-
-    VFSFileProvider testInstance = createTestInstance();
-
-    // TEST - simple negative tests before "pvfs://domain"
-    assertFalse( testInstance.isConnectionRoot( null ) );
-
-    assertFalse( testInstance.isConnectionRoot( createTestInstance( null ) ) );
-
-    assertFalse( testInstance.isConnectionRoot( createTestInstance( "" ) ) );
-
-    assertFalse( testInstance.isConnectionRoot( createTestInstance( " " ) ) );
-
-    assertFalse( testInstance.isConnectionRoot( createTestInstance( "pvfs" ) ) );
-
-    assertFalse( testInstance.isConnectionRoot( createTestInstance( "pvfs:" ) ) );
-
-    assertFalse( testInstance.isConnectionRoot( createTestInstance( "pvfs:/" ) ) );
-
-    assertFalse( testInstance.isConnectionRoot( createTestInstance( "pvfs://" ) ) );
-
-    assertFalse( testInstance.isConnectionRoot( createTestInstance( "pvfs:///" ) ) );
-
-    // TEST "pvfs://domain"
-    assertTrue( testInstance.isConnectionRoot( createTestInstance( "pvfs://someConnection" ) ) );
-
-    assertTrue( testInstance.isConnectionRoot( createTestInstance( "pvfs://some_Connection" ) ) );
-
-    assertTrue( testInstance.isConnectionRoot( createTestInstance( "pvfs://some-Connection" ) ) );
-
-    assertTrue( testInstance.isConnectionRoot( createTestInstance( "pvfs://someConnection123" ) ) );
-
-    assertTrue( testInstance.isConnectionRoot( createTestInstance( "pvfs://123someConnection123" ) ) );
-
-    assertTrue( testInstance.isConnectionRoot( createTestInstance( "pvfs://someConnection/" ) ) );
-
-    assertTrue( testInstance.isConnectionRoot( createTestInstance( "pvfs://some_Connection/" ) ) );
-
-    assertTrue( testInstance.isConnectionRoot( createTestInstance( "pvfs://some-Connection/" ) ) );
-
-    assertTrue( testInstance.isConnectionRoot( createTestInstance( "pvfs://someConnection123/" ) ) );
-
-    assertTrue( testInstance.isConnectionRoot( createTestInstance( "pvfs://123someConnection123/" ) ) );
-
-    assertTrue( testInstance.isConnectionRoot( createTestInstance( "pvfs://Special Character name &#! <>/" ) ) );
-
-    assertTrue( testInstance.isConnectionRoot( createTestInstance( "pvfs://Special Character name &#! <>" ) ) );
-
-    // TEST now we have past the root domain
-    assertFalse( testInstance.isConnectionRoot( createTestInstance( "pvfs://someConnection/someFolderA" ) ) );
-
-    assertFalse( testInstance.isConnectionRoot(
-        createTestInstance( "pvfs://someConnection/someFolderA/directory2" ) ) );
-
-    assertFalse( testInstance.isConnectionRoot(
-        createTestInstance( "pvfs://someConnection/someFolderA/directory2/randomFileC.txt" ) ) );
-
-    assertFalse( testInstance.isConnectionRoot(
-      createTestInstance( "pvfs://Special Character name &#! <>/someFolderA/directory2/randomFileC.txt" ) ) );
-  }
-
-  public void testHasBuckets() {
```

Comment:
These tests were integrated into `VFSConnectionManagerHelperTest`, for `useBuckets(.)`.